src/main/resources/javascript/locales/de.json

```diff
@@ -82,7 +82,10 @@
             "clickHere": "klicken Sie hier.",
             "dialogTitle": "In Arbeit (WIP)",
             "dialogSubTitle": "Markieren Sie Ihren Content als Work in Progress wenn Sie die Veröffentlichung des Contents verhindern wollen. Um mehr über das Work in Progress Feature zu erfahren, ",
-            "label": "Als \"In Arbeit\" markieren",
+            "label" : {
+              "mark": "Als \"In Arbeit\" markieren",
+              "unmark": "Nicht als \"In Arbeit\" markieren"
```

Comment:
is it normal \" ?

Reply:
No idea, Google provides me that translation

Reply:
oh .. you were talking about the marks .. 
I used the previous one